src/Grpc.AspNetCore.Server/GrpcServiceDefinitionRegistry.cs

```diff
@@ -0,0 +1,54 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Grpc.Core;
+
+namespace Grpc.AspNetCore.Server
+{
+    public class GrpcServiceDefinition
+    {
+        public GrpcServiceDefinition(Type serviceType, IList<IMethod> methods)
```

Comment:
nit: make it more obvious that the serviceType is the type of the service implementation class?